parsing/dialects/intermediate/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ValidatorTest.java

```diff
@@ -41,4 +41,11 @@
         + "FROM `ABC` AS `ABC`";
     sql(sql).rewritesTo(expected);
   }
+
+  @Test public void testHostVariable() {
```

Comment:
Are there any restrictions we should be checking regarding where host variables can/cannot occur, what type they have, etc?

Reply:
For now, I'd just like to be able to not alter host variables during validation rewriting. Actually extracting meaningful type information for them would require a larger scope effort (which may or may not be necessary given current needs).

Reply:
The location where host variables can occur should already be enforced during parse time.

Reply:
Ok, SGTM.